docs/TROUBLESHOOTING.md

```diff
@@ -31,15 +31,15 @@ These external services are deployed via [`docker compose`](https://docs.docker.
 Thus, it is worth checking if you are running these services to avoid conflicts when running SI.
 Potentially conflicting services include, but are not limited to, the following:
 
-* PostgreSQL DB
-* OpenTelemetry
+* Jaeger
 * NATS
```

Comment:
should we mention that this is using Jetstream? I think that's the only change!

Reply:
Good call out since, if I recall correctly, they use two separate ports